_includes/pbc/all/install-features/202304.0/install-the-dynamic-store-cms-feature.md

```diff
@@ -0,0 +1,28 @@
+
+This document describes how to integrate the Dynamic Store + CMS feature into a Spryker project.
+
+## Install Feature Frontend
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME | VERSION |  
+| --- | --- | 
+| Spryker Core | {{page.version}} | 
+| CMS | {{page.version}} |
+
+### 1) Set up behavior
+
+If using twig function `renderCmsBlockAsTwig` in twig templates, make sure to provide `storeName` name as a parameter. Otherwise, the function will throw an exception.
```

Comment:
```suggestion
If you are using the twig function `renderCmsBlockAsTwig` in twig templates, make sure to provide `storeName` name as a parameter. Otherwise, the function will throw an exception.
```